openslides_backend/action/generics/create.py

```diff
@@ -50,3 +52,47 @@ def create_write_requests(
 
         response_info: ActionResponseResultsElement = {"id": fqid.id}
         yield response_info
+
+    def validate_required_fields(self, instance: Dict[str, Any]) -> None:
+        """
+        Raises a RequiredFieldsException, if you try to set a required field empty in an update
+        """
+        required_fields = [
+            field.own_field_name
+            for field in self.model.get_required_fields()
+            if field.own_field_name not in instance
+            or (field.own_field_name in instance and not instance[field.own_field_name])
+        ]
+        if required_fields:
+            fqid = FullQualifiedId(self.model.collection, instance["id"])
+            fqid_str = f"Creation of {fqid}"
+            raise RequiredFieldsException(fqid_str, required_fields)
+
+    def concurrent_create_per_execute_other_action(
+        self,
+        instance: Dict[str, Any],
+        action_class: Type["CreateAction"],
+        fields: List[str],
+        additional_dict: Dict[str, Any],
+    ) -> Dict[str, Any]:
+        """
+        This method helps, if there are 2 models that require each other on creation. In the leading model call this method instead the
+        method execute_other_action, to create the instance of the 2. model
+
+        Parameters:
+            instance: instance of the leading model. It's id is used and it's related field(s) are updated to the id of the created related model
+            action_class: The action class to create the related model
+            fields: List of fieldnames of the related class to be created with the id of the leading model
+            additional_dict: Dict of all further values you need to create the related instance
+        Returns:
+            The updated instance of the leading model
+        """
+        payload = {field: instance["id"] for field in fields}
+        payload.update(additional_dict)
+        self.execute_other_action(action_class, [payload])
+        fqid = FullQualifiedId(self.model.collection, instance["id"])
+        for wr in self.write_requests:
+            for event in wr.events:  # type: ignore
+                if event["fqid"] == fqid and event["type"] == EventType.Update:
+                    instance.update(event["fields"])  # type: ignore
+        return instance
```

Comment:
Mmh, I'm not a fan of this - it complicates the process of delegating actions additionally. We talked about checking the resulting write request events, did you try to take that approach?

Reply:
replaced by the existing solution CreateActionWithDependencies

Reply:
The validation took place by checking the resulting write request events